### PR TITLE
test: add a check about the connection description

### PR DESCRIPTION
**Pull request by yunmingyang**

### CODE REVIEW

File: test/check-machines-lifecycle [hunk 1]
```diff
@@ -89,6 +89,21 @@ class TestMachinesLifecycle(VirtualMachinesCase):
         self.waitPageInit()
         self.waitVmRow("subVmTest1")
 
+        # Check connection interpretation
+        b.click("#import-existing-vm")
+        b.wait_visible("#create-vm-dialog")
+        b.wait_visible('#create-vm-dialog button[aria-label="more info"]')
```
Comment (martinpitt) [inline test/check-machines-lifecycle:95]: Would it make sense to actually check what it does, i.e. that hovering over it creates a popup with some significant word or two in it?

**Review by martinpitt — COMMENTED** [review]
Thanks! I triggered tests. I leave the main review to @KKoukiou or @skobyda , but one suggestion.

File: test/check-machines-lifecycle (hunk 1 above)
Comment (yunmingyang) [reply test/check-machines-lifecycle:95]: Make sense, I will complement the case, thanks

**Review by martinpitt — APPROVED** [review]
Thanks! @skobyda , any other comments?

**Review by skobyda — APPROVED** [review]
LGTM. Once the tests pass (apart from that fedora-rawhide which is underlated failure), let's merge this

**Pull request merged**